x-pack/solutions/observability/plugins/slo/server/lib/rules/slo_burn_rate/executor.ts

```diff
@@ -168,17 +167,17 @@ export const getRuleExecutor = (basePath: IBasePath) =>
             actionGroup,
             state: {
               alertState: AlertStates.ALERT,
-              grouping: groupings,
             },
             payload: {
               [ALERT_REASON]: reason,
               [ALERT_EVALUATION_THRESHOLD]: windowDef.burnRateThreshold,
               [ALERT_EVALUATION_VALUE]: Math.min(longWindowBurnRate, shortWindowBurnRate),
               [ALERT_GROUP]: groups,
+              [ALERT_GROUPING]: groupings, // Object, example: { host: { name: 'host-0' } }
               [SLO_ID_FIELD]: slo.id,
               [SLO_REVISION_FIELD]: slo.revision,
               [SLO_INSTANCE_ID_FIELD]: instanceId,
-              ...getEcsGroups(groups),
+              ...getEcsGroupings(groupings),
```

Comment:
could we use `groupingsFlattened` instead of `groupings` so that we don't need to flatten it again in `getEcsGroupings` function?

Reply:
Yep I already talked with @cesco-f about that. I will use `groupingsFlattened` instead of `groupings`

Reply:
Updated code uses now groupingsFlattened instead of groupings `getEcsGroupsFromFlattenGrouping(groupingsFlattened)`